state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -272,6 +272,7 @@ pub mod pallet {
 			origin: SwapOrigin,
 			swap_type: SwapType,
 			broker_commission: Option<AssetAmount>,
+			ingress_fee: Option<AssetAmount>,
```

Comment:
A comment as to why this is optional would be helpful. It can be None when there's a contract initiated swap.